packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -35,9 +37,19 @@
       int id,
       Map<String, Object> params,
       final View containerView) {
-    webView = new InputAwareWebView(context, containerView);
+    
+    Context activityContext = context;
+    Context appContext = context.getApplicationContext();
+    if (appContext instanceof FlutterApplication) {
+      Activity currentActivity = ((FlutterApplication) appContext).getCurrentActivity();
+      if (currentActivity != null) {
+        activityContext = currentActivity;
+      }
+    }
+
+    webView = new InputAwareWebView(activityContext, containerView);
```

Comment:
We can't do that, the context given to platform views is the context for the relevant virtual display.
There are multiple things that can go wrong, one of them is the webview thinking it is in a bigger window than the one it is in.

Reply:
Hello amirh,

my problem is that when deploying a dropdown html on android the application crash

Reply:
thanks, can you please file an issue with a minimal sample app that reproduces the issue?

Reply:
Hello amirh,

When the last update, my issue is about showing a select on android, the app crash.

Exception:
`I/BufferQueue(27514): [unnamed-27514-0](this:0x7f95475c00,id:0,api:0,p:-1,c:-1) BufferQueue core=(27514:io.flutter.plugins.webviewflutterexample)
D/BufferQueueDump(27514): [unnamed-27514-0] this:0x7fae917c80, value:0x7fdcf2f45c, iLen:6
I/BufferQueueConsumer(27514): [unnamed-27514-0](this:0x7f95475c00,id:0,api:0,p:-1,c:27514) connect(C): consumer=(27514:io.flutter.plugins.webviewflutterexample) controlledByApp=true
D/BufferQueueDump(27514): [unnamed-27514-0] this:0x7fae917c80, value:0x7fdcf2f47c, iLen:6
I/BufferQueueConsumer(27514): [unnamed-27514-0](this:0x7f95475c00,id:0,api:0,p:-1,c:27514) setConsumerName: unnamed-27514-0
D/BufferQueueDump(27514): [SurfaceTexture-0-27514-0] this:0x7fae917c80, value:0x7fdcf2f4ec, iLen:6
I/BufferQueueConsumer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:0,p:-1,c:27514) setConsumerName: SurfaceTexture-0-27514-0
I/GLConsumer(27514): [SurfaceTexture-0-27514-0] detachFromContext
I/BufferQueueConsumer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:0,p:-1,c:27514) setDefaultBufferSize: width=480 height=734
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) connect(P): api=1 producer=(247:???) producerControlledByApp=false
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) new GraphicBuffer needed
I/[MALI][Gralloc](27514): [+]r_hnd(0x7fa53b6540), client(50), share_fd(62)
D/GraphicBuffer(27514): register, handle(0x7fa53b6540) (w:480 h:734 s:480 f:0x22 u:0x000900)
I/WebViewFactory(27514): Loading com.google.android.webview version 70.0.3538.80 (code 353808050)
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 0
I/System  (27514): FinalizerDaemon: finalize objects = 1
I/art     (27514): Rejecting re-init on previously-failed class java.lang.Class<hs>
I/art     (27514): Rejecting re-init on previously-failed class java.lang.Class<hs>
I/cr_LibraryLoader(27514): Time to load native libraries: 30 ms (timestamps 382-412)
D/WebView (27514): WebView<init>
I/chromium(27514): [INFO:library_loader_hooks.cc(36)] Chromium logging enabled: level = 0, default verbosity = 0
I/cr_LibraryLoader(27514): Expected native library version number "70.0.3538.80", actual native library version number "70.0.3538.80"
I/[MALI][Gralloc](27514): [+]r_hnd(0x7f8fcc50c0), client(50), share_fd(69)
D/GraphicBuffer(27514): register, handle(0x7f8fcc50c0) (w:480 h:854 s:480 f:0x1 u:0x000b00)
I/cr_BrowserStartup(27514): Initializing chromium process, singleProcess=true
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 0
D/MPlugin (27514): createInstance(): Begin = 8361263
D/MPlugin (27514): create Instance (two parameters) with : com.mediatek.common.telephony.IOnlyOwnerSimSupport
D/MPlugin (27514): Clazz exists on mapping table : false
E/MPlugin (27514): Unsupported class: com.mediatek.common.telephony.IOnlyOwnerSimSupport
D/MPlugin (27514): createInstance(): End = 8361263
D/WebView (27514): addJavascriptInterface=Toaster
D/WebView (27514): loadUrl=https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select
W/cr_media(27514): Requires BLUETOOTH permission
E/libEGL  (27514): validate_display:255 error 3008 (EGL_BAD_DISPLAY)
D/WindowClient(27514): Add to mViews: com.android.internal.policy.PhoneWindow$DecorView{2c8e6a7 V.E...... R.....ID 0,0-0,0}, this = android.view.WindowManagerGlobal@d7109a0
D/OpenGLRenderer(27514): CanvasContext() 0x7fad8b7800
D/ViewRootImpl(27514): hardware acceleration is enabled, this = ViewRoot{db47b4a io.flutter.plugins.webviewflutterexample/io.flutter.plugins.webviewflutterexample.MainActivity,ident = 1}
W/AudioCapabilities(27514): Unsupported mime audio/mpeg-L2
W/AudioCapabilities(27514): Unsupported mime audio/x-adpcm-ms
W/AudioCapabilities(27514): Unsupported mime audio/x-adpcm-dvi-ima
W/AudioCapabilities(27514): Unsupported mime audio/ape
W/AudioCapabilities(27514): Unsupported mime audio/alac
W/AudioCapabilities(27514): Unsupported mime audio/x-adpcm-ms
D/SettingsInterface(27514):  from settings cache , name = accessibility_captioning_enabled , value = null
D/SettingsInterface(27514):  from settings cache , name = accessibility_captioning_font_scale , value = null
W/AudioCapabilities(27514): Unsupported mime audio/x-adpcm-dvi-ima
D/SettingsInterface(27514):  from settings cache , name = accessibility_captioning_locale , value = null
D/SettingsInterface(27514):  from settings cache , name = accessibility_captioning_preset , value = null
V/InputMethodManager(27514): Not IME target window, ignoring
W/VideoCapabilities(27514): Unrecognized profile/level 32768/2 for video/mp4v-es
D/SettingsInterface(27514):  from settings cache , name = accessibility_captioning_enabled , value = null
D/SettingsInterface(27514):  from settings cache , name = accessibility_captioning_font_scale , value = null
D/SettingsInterface(27514):  from settings cache , name = accessibility_captioning_locale , value = null
D/SettingsInterface(27514):  from settings cache , name = accessibility_captioning_preset , value = null
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 0
W/VideoCapabilities(27514): Unsupported mime video/x-ms-wmv
W/VideoCapabilities(27514): Unsupported mime video/divx
W/VideoCapabilities(27514): Unsupported mime video/divx3
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 0
W/VideoCapabilities(27514): Unsupported mime video/xvid
W/VideoCapabilities(27514): Unsupported mime video/flv1
D/Surface (27514): Surface::allocateBuffers(this=0x7f916b3800)
D/OpenGLRenderer(27514): CanvasContext() 0x7fad8b7800 initialize window=0x7f916b3810, title=io.flutter.plugins.webviewflutterexample/io.flutter.plugins.webviewflutterexample.MainActivity
D/Surface (27514): Surface::connect(this=0x7f916b3800,api=1)
W/libEGL  (27514): [ANDROID_RECORDABLE] format: 1
D/mali_winsys(27514): new_window_surface returns 0x3000
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 0
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 0
D/libc-netbsd(27514): [getaddrinfo]: hostname=www.w3schools.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
I/[MALI][Gralloc](27514): [+]r_hnd(0x7fa98215e0), client(50), share_fd(118)
D/GraphicBuffer(27514): register, handle(0x7fa98215e0) (w:480 h:734 s:480 f:0x1 u:0x000b00)
D/libc-netbsd(27514): getaddrinfo: www.w3schools.com get result from proxy gai_error = 0
I/[MALI][Gralloc](27514): [+]r_hnd(0x7f8dbfd580), client(50), share_fd(121)
D/GraphicBuffer(27514): register, handle(0x7f8dbfd580) (w:480 h:734 s:480 f:0x1 u:0x000b00)
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
I/VideoCapabilities(27514): Unsupported profile 4 for video/mp4v-es
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 0
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
W/cr_MediaCodecUtil(27514): Codec OMX.MTK.VIDEO.ENCODER.AVC is disabled due to SDK version 23
W/cr_MediaCodecUtil(27514): HW encoder for video/avc is not available on this device.
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) new GraphicBuffer needed
I/[MALI][Gralloc](27514): [+]r_hnd(0x7fa53b64a0), client(50), share_fd(128)
D/GraphicBuffer(27514): register, handle(0x7fa53b64a0) (w:480 h:734 s:480 f:0x22 u:0x000900)
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/[MALI][Gralloc](27514): [+]r_hnd(0x7f8dbfdb20), client(50), share_fd(131)
D/GraphicBuffer(27514): register, handle(0x7f8dbfdb20) (w:480 h:854 s:480 f:0x1 u:0x000b00)
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- setFrame, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
I/SurfaceView(27514): updateWindow -- OnPreDrawListener, mHaveFrame = true, this = io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854}
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
D/libc-netbsd(27514): [getaddrinfo]: hostname=fonts.googleapis.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: fonts.googleapis.com get result from proxy gai_error = 0
D/libc-netbsd(27514): [getaddrinfo]: hostname=snigelweb-com.videoplayerhub.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): [getaddrinfo]: hostname=static.h-bid.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: static.h-bid.com get result from proxy gai_error = 0
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/libc-netbsd(27514): getaddrinfo: snigelweb-com.videoplayerhub.com get result from proxy gai_error = 0
I/GLConsumer(27514): [SurfaceTexture-0-27514-0] attachToContext
I/[MALI][Gralloc](27514): [+]r_hnd(0x7f8fcc4c60), client(50), share_fd(127)
D/GraphicBuffer(27514): register, handle(0x7f8fcc4c60) (w:480 h:854 s:480 f:0x1 u:0x000b00)
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/libc-netbsd(27514): [getaddrinfo]: hostname=apis.google.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: apis.google.com get result from proxy gai_error = 0
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/libc-netbsd(27514): [getaddrinfo]: hostname=www.google-analytics.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: www.google-analytics.com get result from proxy gai_error = 0
D/libc-netbsd(27514): [getaddrinfo]: hostname=www.googletagservices.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: www.googletagservices.com get result from proxy gai_error = 0
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: fps=0.74 dur=1344.38 max=1344.38 min=1344.38
I/[MALI][Gralloc](27514): [+]r_hnd(0x7f8dbfdbc0), client(50), share_fd(71)
D/GraphicBuffer(27514): register, handle(0x7f8dbfdbc0) (w:480 h:734 s:480 f:0x1 u:0x000b00)
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) new GraphicBuffer needed
I/[MALI][Gralloc](27514): [+]r_hnd(0x7f8dbfdc60), client(50), share_fd(153)
D/GraphicBuffer(27514): register, handle(0x7f8dbfdc60) (w:480 h:734 s:480 f:0x1 u:0x000b00)
I/[MALI][Gralloc](27514): [+]r_hnd(0x7f93c44300), client(50), share_fd(155)
D/GraphicBuffer(27514): register, handle(0x7f93c44300) (w:480 h:734 s:480 f:0x22 u:0x000900)
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 1 is dropped, handle=0x7fa53b64a0
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 2 is dropped, handle=0x7f93c44300
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 1 is dropped, handle=0x7fa53b64a0
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 2 is dropped, handle=0x7f93c44300
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 1 is dropped, handle=0x7fa53b64a0
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 2 is dropped, handle=0x7f93c44300
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 1 is dropped, handle=0x7fa53b64a0
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 2 is dropped, handle=0x7f93c44300
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 1 is dropped, handle=0x7fa53b64a0
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 2 is dropped, handle=0x7f93c44300
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 1 is dropped, handle=0x7fa53b64a0
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 2 is dropped, handle=0x7f93c44300
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 1 is dropped, handle=0x7fa53b64a0
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: slot 2 is dropped, handle=0x7f93c44300
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 2
D/libc-netbsd(27514): [getaddrinfo]: hostname=fonts.gstatic.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: fonts.gstatic.com get result from proxy gai_error = 0
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 0
I/chromium(27514): [INFO:CONSOLE(13)] "Video gallery initializing", source: https://snigelweb-com.videoplayerhub.com/videoloader.js (13)
D/libc-netbsd(27514): [getaddrinfo]: hostname=geo.us.h-bid.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: geo.us.h-bid.com get result from proxy gai_error = 0
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 0
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 0
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: fps=11.72 dur=1364.84 max=557.39 min=13.57
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/libc-netbsd(27514): [getaddrinfo]: hostname=stats.g.doubleclick.net; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: stats.g.doubleclick.net get result from proxy gai_error = 0
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/libc-netbsd(27514): [getaddrinfo]: hostname=adservice.google.es; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: adservice.google.es get result from proxy gai_error = 0
D/libc-netbsd(27514): [getaddrinfo]: hostname=adservice.google.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): [getaddrinfo]: hostname=securepubads.g.doubleclick.net; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: adservice.google.com get result from proxy gai_error = 0
D/libc-netbsd(27514): getaddrinfo: securepubads.g.doubleclick.net get result from proxy gai_error = 0
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/libc-netbsd(27514): [getaddrinfo]: hostname=www.google.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: www.google.com get result from proxy gai_error = 0
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 2
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 2
D/libc-netbsd(27514): [getaddrinfo]: hostname=www.google.es; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: www.google.es get result from proxy gai_error = 0
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 2
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 2
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 2
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/libc-netbsd(27514): [getaddrinfo]: hostname=csi.gstatic.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: csi.gstatic.com get result from proxy gai_error = 0
D/libc-netbsd(27514): [getaddrinfo]: hostname=accounts.google.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: accounts.google.com get result from proxy gai_error = 0
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
I/chromium(27514): [INFO:CONSOLE(65)] "[snhb](3.62s):", source: https://static.h-bid.com/w3schools.com/20190327/snhb-w3schools.com.min.js?20190327 (65)
I/chromium(27514): [INFO:CONSOLE(65)] "[snhb](3.622s):", source: https://static.h-bid.com/w3schools.com/20190327/snhb-w3schools.com.min.js?20190327 (65)
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
I/chromium(27514): [INFO:CONSOLE(65)] "[snhb](4.032s):", source: https://static.h-bid.com/w3schools.com/20190327/snhb-w3schools.com.min.js?20190327 (65)
I/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) queueBuffer: fps=0.83 dur=2423.35 max=1900.94 min=522.41
V/InputMethodManager(27514): START INPUT: io.flutter.view.FlutterView{61e9746 VFE...... .F...... 0,0-480,854} ic=null tba=android.view.inputmethod.EditorInfo@9c18e12 controlFlags=#0
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 1
D/libc-netbsd(27514): [getaddrinfo]: hostname=ice.360yield.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): [getaddrinfo]: hostname=bidder.criteo.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): [getaddrinfo]: hostname=prg.smartadserver.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: bidder.criteo.com get result from proxy gai_error = 0
D/libc-netbsd(27514): getaddrinfo: ice.360yield.com get result from proxy gai_error = 0
D/libc-netbsd(27514): getaddrinfo: prg.smartadserver.com get result from proxy gai_error = 0
D/libc-netbsd(27514): [getaddrinfo]: hostname=ib.adnxs.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: ib.adnxs.com get result from proxy gai_error = 0
D/libc-netbsd(27514): [getaddrinfo]: hostname=fastlane.rubiconproject.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: fastlane.rubiconproject.com get result from proxy gai_error = 0
D/libc-netbsd(27514): [getaddrinfo]: hostname=as-sec.casalemedia.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): [getaddrinfo]: hostname=i.connectad.io; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: as-sec.casalemedia.com get result from proxy gai_error = 0
D/libc-netbsd(27514): getaddrinfo: i.connectad.io get result from proxy gai_error = 0
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/libc-netbsd(27514): [getaddrinfo]: hostname=ssl.gstatic.com; servname=(null); netid=0; mark=0
D/libc-netbsd(27514): [getaddrinfo]: ai_addrlen=0; ai_canonname=(null); ai_flags=1024; ai_family=0
D/libc-netbsd(27514): getaddrinfo: ssl.gstatic.com get result from proxy gai_error = 0
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/BufferQueueProducer(27514): [SurfaceTexture-0-27514-0](this:0x7f95475c00,id:0,api:1,p:247,c:27514) cancelBuffer: slot 2
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
W/System.err(27514): java.lang.ClassCastException: $Proxy0 cannot be cast to android.view.WindowManagerImpl
W/System.err(27514): 	at android.view.Window.setWindowManager(Window.java:575)
W/System.err(27514): 	at android.view.Window.setWindowManager(Window.java:556)
W/System.err(27514): 	at android.app.Dialog.<init>(Dialog.java:192)
W/System.err(27514): 	at android.app.AlertDialog.<init>(AlertDialog.java:200)
W/System.err(27514): 	at android.app.AlertDialog$Builder.create(AlertDialog.java:1086)
W/System.err(27514): 	at GW.<init>(PG:6)
W/System.err(27514): 	at org.chromium.content.browser.input.SelectPopup.show(PG:41)
W/System.err(27514): 	at android.os.MessageQueue.nativePollOnce(Native Method)
W/System.err(27514): 	at android.os.MessageQueue.next(MessageQueue.java:328)
W/System.err(27514): 	at android.os.Looper.loop(Looper.java:164)
W/System.err(27514): 	at android.app.ActivityThread.main(ActivityThread.java:5728)
W/System.err(27514): 	at java.lang.reflect.Method.invoke(Native Method)
W/System.err(27514): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:789)
W/System.err(27514): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:679)
F/chromium(27514): [FATAL:jni_android.cc(249)] Please include Java exception stack in crash report
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
D/OpenSSLLib(27514): OpensslErr:Module:12(116:176); file:external/boringssl/src/crypto/asn1/asn1_lib.c ;Line:186;Function:ASN1_get_object
W/google-breakpad(27514): Failed to generate minidump.
F/google-breakpad(27737): -----BEGIN BREAKPAD MICRODUMP-----
F/google-breakpad(27737): V AndroidWebView:70.0.3538.80
F/google-breakpad(27737): O A arm64 04 aarch64 EnergySistem/angus/angus:6.0/MRA58K/1478765653:user/release-keys
F/google-breakpad(27737): P webview
F/google-breakpad(27737): R 00000005 SIGTRAP 0000007F9CB6DFE8
F/google-breakpad(27737): G 
F/google-breakpad(27737): S 0 0000007FDCF2F2C0 0000007FDCF2F000 0000000000003000
F/google-breakpad(27737): S 0000007FDCF2F000 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
F/google-breakpad(27737): S 0000007FDCF2F180 0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000170000000000000061000000000000005000000000000000EDACEF0DEDACEF0D60F5F2DC7F000000C0F4F2DC7F000000A0F4F2DC7F000000EDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF2F300 EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D00F3F2DC7F000000B47DBD9C7F000000C0F7F2DC7F000000C0F7F2DC7F000000D0F3F2DC7F000000B40DB99B7F000000C1F7F2DC7F000000E03C38B27F0000001600000000000000D8F7F2DC7F000000C0F7F2DC7F0000000000000000000000F0F3F2DC7F000000F00DD99B7F000000000000000000000000000000000000000000000000000000C0F7F2DC7F00000080F7F2DC7F000000320000000000000000F4F2DC7F0000002C0DD99B7F00000070F4F2DC7F000000DC0BD99B7F00000030F5F2DC7F000000F0DB37B27F000000EDACEF0DEDACEF0D6000000000000000000000000000000000000000000000002F00000000000000EDACEF0DEDACEF0D2F000000000000006000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DB0F4F2DC7F000000C80AD99B7F000000
F/google-breakpad(27737): S 0000007FDCF2F480 50F5F2DC7F000000A8DB37B27F000000C0F7F2DC7F0000002F00000000000000000000000000000000000000000000002F00000000000000EDACEF0DEDACEF0D2F000000000000006000000000000000EDACEF0DEDACEF0D20000000000000001C00000000000000F0F9F2DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D7000000000000000C001000000000000E8F7F2DC7F000000200000000000000000F6F2DC7F0000004848B29B7F00000050F5F2DC7F000000B47DBD9C7F000000C0F7F2DC7F000000C0F7F2DC7F000000B0F5F2DC7F000000E40DB99B7F000000EDACEF0DEDACEF0D00000000000000002F00000000000000D8F7F2DC7F000000C0F7F2DC7F0000000000000000000000D0F5F2DC7F000000F00DD99B7F000000000000000000000000000000000000000000000000000000C0F7F2DC7F00000080F7F2DC7F0000002000000000000000E0F5F2DC7F0000002C0DD99B7F00000050F6F2DC7F000000DC0BD99B7F00000030F6F2DC7F000000EDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF2F600 C0010000000000007000000000000000EDACEF0DEDACEF0D000000000000000020000000000000001300000000000000130000000000000080F7F2DC7F0000003300000000000000330000000000000090F6F2DC7F000000C80AD99B7F000000000000000000000024DCB69C7F00000033000000000000007196619B7F000000E8F7F2DC7F00000080F7F2DC7F00000000F7F2DC7F0000008463B59B7F00000080F7F2DC7F0000000000000000000000C0F7F2DC7F00000080F7F2DC7F000000EDACEF0DEDACEF0D90E9719B7F0000007196619B7F00000080F7F2DC7F000000A496619B7F000000E8F7F2DC7F00000078F7F2DC7F00000078F7F2DC7F00000000000000000000002C62B59B7F000000C0010000000000007000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DF0F9F2DC7F000000EDACEF0DEDACEF0D70F7F2DC7F000000EDACEF0DEDACEF0D310000000000000070F7F2DC7F000000D0F8F2DC7F0000009491B59C7F0000000300000000000000EDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF2F780 EDACEF0DEDACEF0DEDACEF0DEDACEF0D000000000000000000000000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D61000000000000005F00000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D06000000000000000000000000000000000000000000000080F7F2DC7F000000EDACEF0DEDACEF0D0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000020000000000000001C000000000000007DE9719B7F000000EDACEF0DEDACEF0D0000000000000000C00100000000000080F8F2DC7F000000000000000000000000000000000000000100000000000000EDACEF0DEDACEF0DE0F9F2DC7F00000068D83D9C7F0000002100000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF2F900 EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D00000000000000002900000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D2500000000000000EDACEF0DEDACEF0D2100000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D000000000000000014437AB27F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D00000000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D50FAF2DC7F000000EDACEF0DEDACEF0D20FAF2DC7F0000003C66229C7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DC0FEF2DC7F0000002C35229C7F00000000000000000000009C1B3FB27F0000000300000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF2FA80 EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D14437AB27F0000000100000000000000EDACEF0DEDACEF0DE0FBF2DC7F000000F0DB37B27F000000EDACEF0DEDACEF0D1800000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D28FCF2DC7F000000EDACEF0DEDACEF0D20FBF2DC7F0000003813BB9C7F000000EDACEF0DEDACEF0D14437AB27F00000040FBF2DC7F0000000013BB9C7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D90FBF2DC7F0000005848B49B7F000000EDACEF0DEDACEF0DB47DBD9C7F00000028FCF2DC7F000000EDACEF0DEDACEF0D00FCF2DC7F0000007808FD9C7F000000EDACEF0DEDACEF0D00000000000000000100000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D010000000000000028FDF2DC7F0000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DC0FCF2DC7F000000A8DB37B27F000000
F/google-breakpad(27737): S 0000007FDCF2FC00 EDACEF0DEDACEF0DEDACEF0DEDACEF0D000000000000000048FEF2DC7F0000000100000000000000EDACEF0DEDACEF0D50FDF2DC7F000000F0DB37B27F000000EDACEF0DEDACEF0D0800000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D00000000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D14437AB27F000000EDACEF0DEDACEF0D00000000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D14437AB27F00000010FDF2DC7F0000009444FD9C7F00000010FDF2DC7F0000000000000000000000EDACEF0DEDACEF0D010000000000000048FEF2DC7F0000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DA0FDF2DC7F0000009C93FC9C7F000000080000000000000000000000000000000000000000000000EDACEF0DEDACEF0D0000000000000000EDACEF0DEDACEF0D70FDF2DC7F000000B47DBD9C7F000000EDACEF0DEDACEF0D08FEF2DC7F000000A0FDF2DC7F000000F017B79C7F000000
F/google-breakpad(27737): S 0000007FDCF2FD80 EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D70FEF2DC7F000000F4D4B49C7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DE0FDF2DC7F000000BC18B69B7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D00FEF2DC7F0000008C18B69B7F000000EDACEF0DEDACEF0D000000000000000020FEF2DC7F00000030ABB69B7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D50FEF2DC7F000000943E319C7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DC0FEF2DC7F000000205C319C7F000000EDACEF0DEDACEF0D000000000000000090FEF2DC7F000000E8A4B59B7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D14437AB27F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DE0FEF2DC7F0000004C8EFB9C7F00000000000000000000005800F3DC7F00000070FFF2DC7F0000002C31B69C7F000000EDACEF0DEDACEF0DB08DFB9C7F000000
F/google-breakpad(27737): S 0000007FDCF2FF00 F04AFC9C7F000000F09BFB9C7F0000004071559E7F000000F04AFC9C7F000000EDACEF0DEDACEF0D0000000000000000D0FFF2DC7F0000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DB08DFB9C7F000000AF6E709B7F0000005800F3DC7F000000EDACEF0DEDACEF0D4000F3DC7F0000001425B79C7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D01000000000000004E00000000000000EDACEF0DEDACEF0D0000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D4000F3DC7F0000004436B79C7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D01000000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D01000000000000004E00000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D5002F3DC7F0000009428B79C7F000000EDACEF0DEDACEF0D00000000000000006BEB669B7F000000AF6E709B7F000000EDACEF0DEDACEF0DB08DFB9C7F000000
F/google-breakpad(27737): S 0000007FDCF30080 0000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DF04AFC9C7F000000F09BFB9C7F0000004071559E7F000000F04AFC9C7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DE8030000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D8CE2939D7F0000004002F3DC7F000000F0DB37B27F000000A00000000000000000000000000000006BEB669B7F000000AF6E709B7F000000EDACEF0DEDACEF0DB08DFB9C7F0000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DF04AFC9C7F000000F09BFB9C7F0000004071559E7F000000F04AFC9C7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D010000000000000001000000000000008802F3DC7F000000E001F3DC7F000000CC4B7AB27F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D1002F3DC7F00000064297AB27F0000000000000000000000EDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF30200 01000000000000008802F3DC7F0000004002F3DC7F000000000000000000000014437AB27F0000004E00000000000000010000000000000001000000000000000100000000000000EDACEF0DEDACEF0D9002F3DC7F000000E83FB79C7F0000009002F3DC7F00000000207AB27F0000000100000000000000EDACEF0DEDACEF0D0000000000000000EDACEF0DEDACEF0DA002F3DC7F000000243EB79C7F000000B002F3DC7F000000205F7AB27F0000006004F3DC7F00000084617AB27F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D000000000000000000000000000000000000000000000000000000000000000000000000000000006809F3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D0000000000000000B010F3DC7F0000008010F3DC7F0000000000000000000000EDACEF0DEDACEF0D4E00000000000000EDACEF0DEDACEF0D03000000000000000000000000000000EDACEF0DEDACEF0D01000000000000004E0000000000000001000000000000001300000000000000
F/google-breakpad(27737): S 0000007FDCF30380 EDACEF0DEDACEF0DF4FE797400000000EDACEF0DEDACEF0DDC0A2D73000000000000000000000000B010F3DC7F0000008010F3DC7F0000006812F3DC7F000000B84285B27F0000000814F3DC7F00000004000000000000000000000000000000EDACEF0DEDACEF0D9C070000000000008004F3DC7F000000301D3FAE7F0000009004F3DC7F000000301D3FAE7F0000000700000000000000141D3FAE7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D0000000000000000A004F3DC7F00000000203FAE7F0000000700000000000000B81F3FAE7F000000A004F3DC7F000000407063B27F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D4CADFE7200000000C807F3DC7F0000009018FC72000000000100000000000000847063B27F000000EDACEF0DEDACEF0D980AF3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D0000000000000000B010F3DC7F0000008010F3DC7F0000006812F3DC7F000000
F/google-breakpad(27737): S 0000007FDCF30500 B84285B27F0000000814F3DC7F000000040000000000000000000000000000000100000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D000000000000000000000000000000006809F3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DC807F3DC7F000000EC7B437400000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D00000000000000000000000000000000B010F3DC7F0000008010F3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D00000000000000000100000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D0100000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D00000000000000006809F3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DC807F3DC7F00000038F8427400000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF30680 EDACEF0DEDACEF0D3C21C37300000000EDACEF0DEDACEF0D6809F3DC7F000000EDACEF0DEDACEF0D00000000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D0100000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D6809F3DC7F0000009009F3DC7F000000874AAC7100000000C807F3DC7F000000B845B57300000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D0000000000000000EDACEF0DEDACEF0D0000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DA808F3DC7F0000006809F3DC7F0000009009F3DC7F0000006809F3DC7F0000009009F3DC7F0000005C343FAE7F0000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D4008F3DC7F000000283140AE7F0000006809F3DC7F000000874AAC71000000004008F3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF30800 EDACEF0DEDACEF0DEDACEF0DEDACEF0DA808F3DC7F0000006809F3DC7F0000009009F3DC7F0000006809F3DC7F0000009009F3DC7F000000874AAC7100000000E008F3DC7F000000284470AE7F000000780AF3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D0100000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D6809F3DC7F000000AC0AF3DC7F000000EDACEF0DEDACEF0D0100000000000000900AF3DC7F000000780DF3DC7F0000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D5008F3DC7F000000EDACEF0DEDACEF0D600AF3DC7F00000010D069AE7F000000EDACEF0DEDACEF0DE4CF69AE7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D01000000000000000000000000000000EDACEF0DEDACEF0DD60B0000000000006F000000000000000300000000000000EDACEF0DEDACEF0DA40AF3DC7F00000000000000000000000000000000000000EDACEF0DEDACEF0D874AAC7100000000
F/google-breakpad(27737): S 0000007FDCF30980 EDACEF0DEDACEF0D9009F3DC7F000000EDACEF0DEDACEF0DB84285B27F0000000814F3DC7F00000004000000000000000000000000000000EDACEF0DEDACEF0D000000000000000000000000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D94683FAE7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D0100000000000000EDACEF0DEDACEF0D0100000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DD60B0000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DB8FA327300000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D0000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D0000000000000000EDACEF0DEDACEF0D0000000000000000B010F3DC7F0000008010F3DC7F0000006812F3DC7F000000B84285B27F0000000814F3DC7F00000004000000000000000000000000000000
F/google-breakpad(27737): S 0000007FDCF30B00 EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D01000000000000000000000000000000EDACEF0DEDACEF0DD60B000000000000EDACEF0DEDACEF0D4423C27400000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DD60B000000000000980CF3DC7F000000EDACEF0DEDACEF0D0000000000000000EDACEF0DEDACEF0D0400000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D0100000000000000EDACEF0DEDACEF0D5430C27400000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D980CF3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D300DF3DC7F000000ACFC46AE7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D780DF3DC7F000000480EF3DC7F000000680EF3DC7F000000487E107200000000EDACEF0DEDACEF0D0000000000000000980CF3DC7F0000005C343FAE7F000000
F/google-breakpad(27737): S 0000007FDCF30C80 0000000000000000EDACEF0DEDACEF0D3810F3DC7F000000100DF3DC7F000000283140AE7F000000480EF3DC7F000000487E107200000000100DF3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D780DF3DC7F000000480EF3DC7F000000680EF3DC7F000000487E107200000000EDACEF0DEDACEF0D0000000000000000B00DF3DC7F0000004C1370AE7F000000680EF3DC7F000000B010F3DC7F000000EDACEF0DEDACEF0D0000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D800FF3DC7F000000487E107200000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D800FF3DC7F0000000000000000000000000000000000000000000000000000003810F3DC7F000000F09982AE7F0000000000000000000000EDACEF0DEDACEF0DB00EF3DC7F0000000CCB61AE7F000000800FF3DC7F000000EDACEF0DEDACEF0D6C0FF3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D8010F3DC7F000000EDACEF0DEDACEF0D389982AE7F000000
F/google-breakpad(27737): S 0000007FDCF30E00 50AD82AE7F000000EDACEF0DEDACEF0D700FF3DC7F000000010000000000000000000000000000008C0FF3DC7F000000E80DF3DC7F000000E00DF3DC7F000000EDACEF0DEDACEF0D0000000000000000487E107200000000EDACEF0DEDACEF0D680EF3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D8C0FF3DC7F000000EDACEF0DEDACEF0D0000000000000000000FF3DC7F000000406D65AE7F000000EDACEF0DEDACEF0D0000000000000000B00FF3DC7F00000000375FB27F0000000400000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D05000000000000003011F3DC7F000000504385B27F000000007069B27F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D0100000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D407769B27F000000487769B27F000000B010F3DC7F000000B010F3DC7F0000008010F3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D80E2DC7100000000EDACEF0DEDACEF0D508B62AE7F000000
F/google-breakpad(27737): S 0000007FDCF30F80 EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D005C5FB27F000000B010F3DC7F000000A45C5FB27F000000B010F3DC7F000000B010F3DC7F0000008010F3DC7F000000EDACEF0DEDACEF0DB010F3DC7F000000B010F3DC7F0000008010F3DC7F000000EDACEF0DEDACEF0D00000000000000000000000000000000EDACEF0DEDACEF0D000000000000000000000000000000000000000000000000EDACEF0DEDACEF0D000000000000000000000000000000000000000000000000EDACEF0DEDACEF0D000000000000000000000000000000000000000000000000EDACEF0DEDACEF0D0000000000000000EDACEF0DEDACEF0D0500000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D0000000000000000EDACEF0DEDACEF0DB011F3DC7F000000CC2F85B27F0000007812F3DC7F000000904085B27F0000003014F3DC7F00000006000000000000007012F3DC7F0000002813F3DC7F000000A812F3DC7F0000008012F3DC7F000000
F/google-breakpad(27737): S 0000007FDCF31100 3014F3DC7F0000006012F3DC7F0000007812F3DC7F000000504385B27F0000003014F3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DFC165EAE7F000000687C5EAE7F000000C4165EAE7F0000008011F3DC7F00000064297AB27F00000002000000000000008012F3DC7F00000001000000000000008012F3DC7F000000B011F3DC7F000000143185B27F000000B011F3DC7F000000742F85B27F0000007812F3DC7F000000EDACEF0DEDACEF0D9013F3DC7F0000006CDD37B27F0000004014F3DC7F0000000814F3DC7F0000000600000000000000302B85B27F0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000EDACEF0DEDACEF0D9D1BF3DC7F000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF31280 EDACEF0DEDACEF0DEDACEF0DEDACEF0D030000000000000000000000000000000800000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D28000000000000000000000000000000100000000000000001000000000000009D1BF3DC7F0000006300000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0D00000000000000000000000000000000EDACEF0DEDACEF0D08000000000000000000000000000000EDACEF0DEDACEF0D6100F3DC7F000000B8793DB27F000000000000000000000000000000000000009013F3DC7F0000005CDD37B27F0000004014F3DC7F0000000814F3DC7F0000000600000000000000302B85B27F0000000000000000000000EDACEF0DEDACEF0DC013F3DC7F000000543285B27F000000000000000000000000000000000000000000000000000000000000000000000000000000000000002CAA80B27F00000000000000000000000014F3DC7F0000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF31400 06000000000000009D1BF3DC7F000000B71BF3DC7F000000C01BF3DC7F000000CC1BF3DC7F000000D51BF3DC7F000000EB1BF3DC7F0000000000000000000000001CF3DC7F0000003D1CF3DC7F000000501CF3DC7F000000651CF3DC7F000000801CF3DC7F000000931CF3DC7F000000AC1CF3DC7F000000C51CF3DC7F000000DF1CF3DC7F000000061FF3DC7F000000A61FF3DC7F000000C51FF3DC7F000000000000000000000000000000000000002100000000000000004086B27F0000001000000000000000FF0000000000000006000000000000000010000000000000110000000000000064000000000000000300000000000000401085B27F00000004000000000000003800000000000000050000000000000009000000000000000700000000000000008080B27F000000080000000000000000000000000000000900000000000000543285B27F0000000B0000000000000000000000000000000C0000000000000000000000000000000D000000000000000000000000000000
F/google-breakpad(27737): S 0000007FDCF31580 0E000000000000000000000000000000170000000000000001000000000000001900000000000000E815F3DC7F0000001F00000000000000DE1FF3DC7F0000000F00000000000000F815F3DC7F000000000000000000000000000000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
F/google-breakpad(27737): S 0000007FDCF31700 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
F/google-breakpad(27737): S 0000007FDCF31880 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
F/google-breakpad(27737): S 0000007FDCF31A00 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
F/google-breakpad(27737): S 0000007FDCF31B80 000000000000000000000000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF31D00 EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D
F/google-breakpad(27737): S 0000007FDCF31E80 EDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0DEDACEF0D0000000000000000
F/google-breakpad(27737): C 060000800000000000F3F2DC7F00000040E609807F0000000004000000000000C0D284B27F000000000000000000000001400000000000000000000000000000849744B27F00000000000000000000000000000000000000849744B27F0000000040000000000000014000000000000000000000000000000100000000000000589744B27F00000018B943B27F0000000000000000000000489744B27F00000078F7F2DC7F00000070F7F2DC7F0000004F0000000000000018E543B27F0000000080679E7F000000F0F9F2DC7F00000070D384B27F000000C9F2F2DC7F000000C8F2F2DC7F000000C00100000000000060F7F2DC7F000000C4DEB69C7F000000C0F2F2DC7F000000E8DFB69C7F0000000000008010000000000000009C40B39B7F000000000000000000000060F5F2DC7F000000C0F4F2DC7F000000CDCCCCCCCCCCEC3F0000000000000000CDCCCCCCCCCC7C4000000000000000007480ADEB0000000000000000000000007580ADEB0000000000000000000000007680ADEB0000000000000000000000007780ADEB00000000000000000000000000757AB27F00000000000000000000005003F3DC7F000000000000000000000088777AB27F00000000000000000000006812F3DC7F0000000000000000000000B84285B27F00000000000000000000000814F3DC7F000000000000000000000004000000000000000000000000000000000000000000000000000000000000008080ADEB0000000000000000000000008180ADEB0000000000000000000000008280ADEB0000000000000000000000008380ADEB0000000000000000000000008480ADEB0000000000000000000000008580ADEB0000000000000000000000008680ADEB0000000000000000000000008780ADEB0000000000000000000000008880ADEB0000000000000000000000008980ADEB0000000000000000000000008A80ADEB0000000000000000000000008B80ADEB0000000000000000000000008C80ADEB0000000000000000000000008D80ADEB0000000000000000000000008E80ADEB0000000000000000000000008F80ADEB000000000000000000000000
F/google-breakpad(27737): M 0000007FB2851000 0000000000000000 0000000000004000 53EE26F6797625810D76B39B8E3802490 app_process64
F/google-breakpad(27737): M 0000000070BEA000 0000000000000000 0000000004ACC000 2B8A574AE5BB5C6F2E10D9ED5E234FFF0 boot.oat
F/google-breakpad(27737): M 0000007F8CBBD000 0000000000000000 00000000006C2000 447E7F6A76EBE3D710D8F9D4CA3FDEB30 base.odex
F/google-breakpad(27737): M 0000007F9726A000 0000000000000000 0000000000003000 518BE4C4FF5C8A21FB5A9EB947673DBB0 libwebviewchromium_plat_support.so
F/google-breakpad(27737): M 0000007F9740D000 0000000000000000 000000000001D000 8C174F42E5A6698F0D20F11524B06BE60 libgui_ext.so
F/google-breakpad(27737): M 0000007F97613000 0000000000000000 0000000000005000 B9AB5039A6FB94919689821A2497E69F0 libperfservicenative.so
F/google-breakpad(27737): M 0000007F984A2000 0000000000000000 000000000000C000 9E26D5F799C9B360BF0CF545A31AF42F0 gralloc.mt6735m.so
F/google-breakpad(27737): M 0000007F999C4000 0000000000D20000 0000000000702000 000000000000000000000000000000000 libflutter.so
F/google-breakpad(27737): M 0000007F9A356000 0000000000000000 000000000034C000 4B5AE91951E01E4ECCA6A305F777AB550 base.odex
F/google-breakpad(27737): M 0000007F9AEEC000 0000000000000000 0000000003784000 D7213DB83F9ACD5AE6E7840E7E98668E0 libwebviewchromium.so
F/google-breakpad(27737): M 0000007FA3AC1000 0000000000000000 0000000000002000 B7013668F8E4C3AD433F94D87F1E0B8D0 libwebviewchromium_loader.so
F/google-breakpad(27737): M 0000007FA3AD4000 0000000000000000 0000000000001000 8A014B89B1F0D20AFA5BBAB622EEAD290 libjnigraphics.so
F/google-breakpad(27737): M 0000007FA3AE6000 0000000000000000 000000000001B000 73571228F72C47DB7AA4B84A25CBF1820 libcompiler_rt.so
F/google-breakpad(27737): M 0000007FA3B79000 0000000000000000 0000000000016000 EF997E175E26C38CBE878E0149B4003B0 libandroid.so
F/google-breakpad(27737): M 0000007FA3BA4000 0000000000000000 0000000000009000 29CEDC4D59BE011B41DA3CB63F27A3360 libgpu_aux.so
F/google-breakpad(27737): M 0000007FA39F5000 0000000000000000 000000000176C000 2EBF17E8F9670B910C85C5379B360A7F0 libGLES_mali.so
F/google-breakpad(27737): M 0000007FA53CC000 0000000000000000 0000000000033000 8BA37F5912D2B367B8EE2E3B958F11090 FwkPlugin.odex
F/google-breakpad(27737): M 0000007FA5441000 0000000000000000 0000000000010000 0E3FACDF8B80AED0DCB8112C78BE5D750 libstagefright_amrnb_common.so
F/google-breakpad(27737): M 0000007FA5458000 0000000000000000 000000000002F000 A7156E52B297844F8877EE7ECA82F1590 libexif.so
F/google-breakpad(27737): M 0000007FA54AB000 0000000000000000 000000000000F000 6534F5C3FEB92CE8472704CAE58B1A0C0 libjhead.so
F/google-breakpad(27737): M 0000007FA54CC000 0000000000000000 000000000001F000 8844F83483C38CD4EEE0E97C99FE7A6E0 libmtp.so
F/google-breakpad(27737): M 0000007FA54F8000 0000000000000000 0000000000071000 0A185D265E238E28E78302C26335B8990 libmedia_jni.so
F/google-breakpad(27737): M 0000007FA6C83000 0000000000000000 0000000000005000 2A1DBCAC3C22DA814CC2E5B37BD164F80 libged.so
F/google-breakpad(27737): M 0000007FA8C44000 0000000000000000 0000000000025000 6587DD6673103E676DE8566EB2414B120 libkeymaster1.so
F/google-breakpad(27737): M 0000007FA9347000 0000000000000000 000000000000B000 BDD4776B7C68FDC96769AADFC2D2159D0 libkeymaster_messages.so
F/google-breakpad(27737): M 0000007FA9362000 0000000000000000 000000000001F000 4E8890F0D463982AA9F1B4F5F95259D90 libsoftkeymasterdevice.so
F/google-breakpad(27737): M 0000007FA9391000 0000000000000000 000000000000F000 9B721322D1F294086BECE917906601EB0 libkeystore_binder.so
F/google-breakpad(27737): M 0000007FA93B3000 0000000000000000 0000000000003000 6A6C6D85441C7952DB2CBB93A1A47F140 libkeystore-engine.so
F/google-breakpad(27737): M 0000007FA93C4000 0000000000000000 0000000000029000 0D7C371F4EA4CB033ADE1ECB03C8F7C20 libjavacrypto.so
F/google-breakpad(27737): M 0000007FA95D3000 0000000000000000 0000000000002000 D08D5CBABA0CB5CFD2C91F3F44C241B10 memtrack.mt6735m.so
F/google-breakpad(27737): M 0000007FAB084000 0000000000000000 0000000000059000 06A1900CA43F9F8984B77099E65A09170 libjavacore.so
F/google-breakpad(27737): M 0000007FAE2EE000 0000000000000000 0000000000001000 6DC4543AC48893C6B2CF78581520308F0 libsigchain.so
F/google-breakpad(27737): M 0000007FAE2D5000 0000000000000000 0000000000610000 79B100026789329BF0346EF517ADAE710 libart.so
F/google-breakpad(27737): M 0000007FAEBE9000 0000000000000000 0000000000056000 38983DE4C75A73EE3516797725363B240 libfs_mgr.so
F/google-breakpad(27737): M 0000007FAEC51000 0000000000000000 0000000000002000 71EBB598FE94374A5C201532FD90B38B0 libnvram_sec.so
F/google-breakpad(27737): M 0000007FAEC64000 0000000000000000 0000000000001000 B6BF9E3013CD5B31B7D1134CBC8A72F90 libnvram_platform.so
F/google-breakpad(27737): M 0000007FAEC76000 0000000000000000 0000000000003000 4F6E9809F080F48ECF1DC8934789BB060 libcustom_nvram.so
F/google-breakpad(27737): M 0000007FAEC98000 0000000000000000 000000000003C000 CC1B27716A5809A26A0CA355CBF8DB390 libmtkjpeg.so
F/google-breakpad(27737): M 0000007FAECE6000 0000000000000000 0000000000014000 1C620DFF68AB5E343BDD43DD12AB7EDD0 libnvram.so
F/google-breakpad(27737): M 0000007FAED0A000 0000000000000000 000000000000D000 2880A2BC92AD53428773075BAC62DDA20 libbwc.so
F/google-breakpad(27737): M 0000007FAED2A000 0000000000000000 0000000000006000 765D1094D6DDD4639EB31C76E60E5CDB0 libSwJpgCodec.so
F/google-breakpad(27737): M 0000007FAED42000 0000000000000000 0000000000006000 02E6CEC16A52FFD65297E9D244242DB60 libnvramagentclient.so
F/google-breakpad(27737): M 0000007FAED5A000 0000000000000000 0000000000004000 60753EDFD6940E0B8B08405613A3E63F0 libdrmmtkwhitelist.so
F/google-breakpad(27737): M 0000007FAECD2000 0000000000000000 0000000000FB9000 78375DD31F27ED1AA5CB8C34EC069D200 libLLVM.so
F/google-breakpad(27737): M 0000007FAFD47000 0000000000000000 000000000004A000 78F4C066F2492BEAC6324EC3F0E4C0AB0 libbcinfo.so
F/google-breakpad(27737): M 0000007FAFDA3000 0000000000000000 0000000000062000 D1B331CDB20CCE936475442798ACFDFC0 libbcc.so
F/google-breakpad(27737): M 0000007FAFE18000 0000000000000000 0000000000014000 97B3CEE8156A99664C3D167A95DE24140 libvcodec_utility.so
F/google-breakpad(27737): M 0000007FAFE44000 0000000000000000 0000000000010000 F4EF32FDE9840ADF844C3B5B6C8903A00 libcommon_time_client.so
F/google-breakpad(27737): M 0000007FAFE68000 0000000000000000 000000000001B000 FDCE9B438EC4175F0A62E71A54D9A4820 libnl.so
F/google-breakpad(27737): M 0000007FAFE96000 0000000000000000 0000000000023000 ED9FF79ECD5EBF6B8E580AA3C23C1DF40 libprotobuf-cpp-lite.so
F/google-breakpad(27737): M 0000007FAFECA000 0000000000000000 0000000000010000 2B1D4AEDBF30C3149E95B7A6C52E6A730 libpq_prot.so
F/google-breakpad(27737): M 0000007FAFEE8000 0000000000000000 0000000000013000 AA0411593C14270DF8608547C34B61660 libpqservice.so
F/google-breakpad(27737): M 0000007FAFF0F000 0000000000000000 0000000000002000 15C19DBF7A088A401CC973C4E064C6920 libion_mtk.so
F/google-breakpad(27737): M 0000007FAFF23000 0000000000000000 0000000000004000 7FE5D4953E19512BBF6B7719555B82430 libm4u.so
F/google-breakpad(27737): M 0000007FAFF38000 0000000000000000 0000000000002000 2762AB5FC15145353DA17C66B425910C0 libion.so
F/google-breakpad(27737): M 0000007FAFF4A000 0000000000000000 0000000000012000 484BBBC9E848B619E806260E5A8BF1120 libJpgDecPipe.so
F/google-breakpad(27737): M 0000007FAFF6E000 0000000000000000 0000000000002000 379A81D879BBADA22EB8BB2B220488EB0 libmrdump.so
F/google-breakpad(27737): M 0000007FAFF7A000 0000000000000000 00000000000C8000 B3243B3F3D8D183108157B1F1CF88F250 libdrmmtkutil.so
F/google-breakpad(27737): M 0000007FB005B000 0000000000000000 0000000000003000 8EB59178F6628CC1720DA8E66599D1D50 libcustom_prop.so
F/google-breakpad(27737): M 0000007FB0073000 0000000000000000 000000000000A000 3F3C58A551F57089404BE16A159622330 libstagefright_avc_common.so
F/google-breakpad(27737): M 0000007FB008E000 0000000000000000 0000000000001000 4854BBD848B685C261AB882D61CA03D00 libstagefright_enc_common.so
F/google-breakpad(27737): M 0000007FB00A0000 0000000000000000 0000000000003000 90B2DA04E934068185CE2259FE48D47B0 libpowermanager.so
F/google-breakpad(27737): M 0000007FB00B5000 0000000000000000 000000000001D000 E506EBA6710DDA0F7CA07D9DD1B7E67E0 libvorbisidec.so
F/google-breakpad(27737): M 0000007FB00E3000 0000000000000000 0000000000003000 F69A1D41501B87BBED8485736998CBAF0 libstagefright_yuv.so
F/google-breakpad(27737): M 0000007FB00F1000 0000000000000000 000000000003E000 722DEE3DA8BEFFC92FF3821147073D180 libstagefright_omx.so
F/google-breakpad(27737): M 0000007FB0145000 0000000000000000 000000000004B000 DF86B2AD5EAF1FAA82563F1858464F550 libopus.so
F/google-breakpad(27737): M 0000007FB01A2000 0000000000000000 0000000000004000 045F26362362A892C17D9B9F9088C3FE0 libmediautils.so
F/google-breakpad(27737): M 0000007FB01B3000 0000000000000000 0000000000023000 FD7CB90AB10EEFB5465FF73A4EB457AF0 libdrmframework.so
F/google-breakpad(27737): M 0000007FB01EB000 0000000000000000 0000000000029000 AC768534C9FD3C46C6DEF268516227990 libRScpp.so
F/google-breakpad(27737): M 0000007FB0224000 0000000000000000 0000000000056000 B6082B3CA21975CFE5A0B078D441C6E90 libRS.so
F/google-breakpad(27737): M 0000007FB028E000 0000000000000000 0000000000004000 913CD0FB61B018EE54C9760C1DDE46590 libmtk_drvb.so
F/google-breakpad(27737): M 0000007FB02A3000 0000000000000000 0000000000001000 8534264CE43163C33F404399DB2F89160 libmtksqlite3_custom.so
F/google-breakpad(27737): M 0000007FB02B5000 0000000000000000 0000000000003000 D29779CC0B536CD74B66B8749DE9B24E0 libspeexresampler.so
F/google-breakpad(27737): M 0000007FB02C9000 0000000000000000 0000000000023000 88E8CC294C1A5A8C43A461D6D66B44960 libvcodecdrv.so
F/google-breakpad(27737): M 0000007FB0326000 0000000000000000 000000000000B000 481B8369664B1406A9B1A0D1D8C305B50 libnbaio.so
F/google-breakpad(27737): M 0000007FB0343000 0000000000000000 000000000000A000 7EDAFD429CC4511D452953A31A1957E80 libsysutils.so
F/google-breakpad(27737): M 0000007FB035F000 0000000000000000 000000000001A000 1ABD0021A693EBDAC4A1FCE6AE89EAD10 libpcre.so
F/google-breakpad(27737): M 0000007FB038A000 0000000000000000 0000000000004000 45B0CB7C733714CD51E4FF2990094CA70 libwpa_client.so
F/google-breakpad(27737): M 0000007FB039C000 0000000000000000 00000000000D0000 42ED363F7861E421A4E76A300FD376A10 libGLES_trace.so
F/google-breakpad(27737): M 0000007FB046C000 0000000000000000 000000000007D000 AE8CCF1B84BF59C459488FF10C0242680 libdpframework.so
F/google-breakpad(27737): M 0000007FB04FD000 0000000000000000 0000000000001000 B1253F3992C4C9B000689CE473DDCDA90 libmhalImageCodec.so
F/google-breakpad(27737): M 0000007FB050D000 0000000000000000 0000000000069000 30AF064872DB83BD258E2C228333C8B90 libft2.so
F/google-breakpad(27737): M 0000007FB058A000 0000000000000000 0000000000037000 D7FC11E9DE59E507B0D50F70AE4B45B90 libpng.so
F/google-breakpad(27737): M 0000007FB05D2000 0000000000000000 0000000000005000 90A15F6BB65925BF70832EDB69BE65320 libtouchfilter.so
F/google-breakpad(27737): M 0000007FB05E8000 0000000000000000 0000000000004000 CDDD1615D532AF297B378BB4580336890 libgralloc_extra.so
F/google-breakpad(27737): M 0000007FB05FB000 0000000000000000 000000000000E000 EC772850D11C9D334D79803D894849D60 libui_ext.so
F/google-breakpad(27737): M 0000007FB061C000 0000000000000000 0000000000001000 FD72BC16CE855289E91F373C1DE2052A0 libsync.so
F/google-breakpad(27737): M 0000007FB062E000 0000000000000000 000000000001A000 1EBEC4BA52013DB276C60485762F454D0 libaed.so
F/google-breakpad(27737): M 0000007FB0662000 0000000000000000 0000000000003000 9270BAA844A896450379061B7C9DE66E0 libstdc++.so
F/google-breakpad(27737): M 0000007FB0676000 0000000000000000 000000000001B000 D4FF69A4636325C008406829B4B0A9960 libunwind.so
F/google-breakpad(27737): M 0000007FB070E000 0000000000000000 0000000000009000 9F65F8AB72869C2ED5FB81B778D014A80 libbase.so
F/google-breakpad(27737): M 0000007FB0728000 0000000000000000 0000000000004000 49DB19102E5933478F5D22217149DDA50 libeffects.so
F/google-breakpad(27737): M 0000007FB073D000 0000000000000000 0000000000002000 7719BEE4A8D2924F06E97DA05A94B0CD0 libstagefright_http_support.so
F/google-breakpad(27737): M 0000007FB074F000 0000000000000000 0000000000023000 9869582E26A7FCEC365A73A822E115BF0 libstagefright_foundation.so
F/google-breakpad(27737): M 0000007FB075E000 0000000000000000 00000000002FD000 447A49B948951B9F243192E938FF98610 libstagefright.so
F/google-breakpad(27737): M 0000007FB0A7F000 0000000000000000 00000000000B4000 EFAECAF07386037FC8FB4D51F8A371970 libhwui.so
F/google-breakpad(27737): M 0000007FB0B4A000 0000000000000000 0000000000009000 DF8FF0C798511D35FC349FBF0C4067C70 libmtkcamera_client.so
F/google-breakpad(27737): M 0000007FB0B65000 0000000000000000 0000000000009000 8C115FD7E73F9D768E1BEEBF78243EFB0 libmtksqlite3_android.so
F/google-breakpad(27737): M 0000007FB0B7B000 0000000000000000 0000000000020000 56BCAD3E0AB914D5FA7BD7AF5E4EA4EC0 libaal.so
F/google-breakpad(27737): M 0000007FB0BA0000 0000000000000000 0000000000002000 4134A98728427B80782F0135F828BDC30 libradio_metadata.so
F/google-breakpad(27737): M 0000007FB0BB3000 0000000000000000 0000000000003000 2D1927D137EB0979010162DBB54DB05E0 libnativebridge.so
F/google-breakpad(27737): M 0000007FB0BC7000 0000000000000000 0000000000002000 84EDCA89446CAAD6AAC2E43E77FA5E470 libprocessgroup.so
F/google-breakpad(27737): M 0000007FB0BDA000 0000000000000000 0000000000016000 4BA05C42DE47CC4F62117D5EBD16060F0 libminikin.so
F/google-breakpad(27737): M 0000007FB0BFE000 0000000000000000 000000000000E000 5F9983CB28F08612471A054AE892BC6C0 libsoundtrigger.so
F/google-breakpad(27737): M 0000007FB0C1D000 0000000000000000 000000000000E000 77EA6AD6508E87BBEB15C113A8B103370 libradio.so
F/google-breakpad(27737): M 0000007FB0C40000 0000000000000000 0000000000002000 9ED151E0BB2D0CF1D90EF3C1C5871AEA0 libnetd_client.so
F/google-breakpad(27737): M 0000007FB0C52000 0000000000000000 0000000000013000 B690718FF5F8926E63D1B56B380D2E7A0 libimg_utils.so
F/google-breakpad(27737): M 0000007FB0C63000 0000000000000000 0000000000439000 806DE9094E37236E8A2637911C5B71570 libpdfium.so
F/google-breakpad(27737): M 0000007FB10C2000 0000000000000000 000000000000A000 C7C613128FEFDE76C5AB6FD8E6FEF5360 libaudioutils.so
F/google-breakpad(27737): M 0000007FB10DE000 0000000000000000 000000000001C000 03390F98F7EE162CEB96432E9E57C9B10 libz.so
F/google-breakpad(27737): M 0000007FB110B000 0000000000000000 000000000006E000 7B7D99A0DBCED8DA36757F7C90BDB7470 libharfbuzz_ng.so
F/google-breakpad(27737): M 0000007FB118B000 0000000000000000 0000000000004000 F0AB66589830E5B9B37BD08EC83720D30 libusbhost.so
F/google-breakpad(27737): M 0000007FB11A0000 0000000000000000 0000000000039000 FB756420085D25D9D57B137B0E5487480 libjpeg.so
F/google-breakpad(27737): M 0000007FB1185000 0000000000000000 000000000014F000 CFB5AA2F22D3560385B2CF7A7DA91B940 libmedia.so
F/google-breakpad(27737): M 0000007FB1305000 0000000000000000 0000000000236000 ED4D1C6C58368381F4EE39568053B2680 libicui18n.so
F/google-breakpad(27737): M 0000007FB1547000 0000000000000000 000000000019C000 7D9F92A9568B2EA2378E381FEDC53E830 libicuuc.so
F/google-breakpad(27737): M 0000007FB170A000 0000000000000000 0000000000043000 2E0FF604DA519245CD1617E9196AF57A0 libssl.so
F/google-breakpad(27737): M 0000007FB174B000 0000000000000000 000000000011B000 2AADF6F0E38AC099B9BEA56A1D7FCF870 libcrypto.so
F/google-breakpad(27737): M 0000007FB188A000 0000000000000000 000000000005A000 D8E83CFF72C65FB9431EF32543BEF0FC0 libsonivox.so
F/google-breakpad(27737): M 0000007FB18F7000 0000000000000000 0000000000015000 85E1E750FF2BB7677C5604C0376B5B1E0 libselinux.so
F/google-breakpad(27737): M 0000007FB191F000 0000000000000000 0000000000007000 A2899BBA55694CBC094B0E8210CA1A870 libhardware_legacy.so
F/google-breakpad(27737): M 0000007FB193A000 0000000000000000 0000000000002000 F90E0BA29E3BC916136C25A2E74B9C640 libhardware.so
F/google-breakpad(27737): M 0000007FB194E000 0000000000000000 0000000000003000 099F249EB2CAE65E1B5EDAE46FD86D570 libETC1.so
F/google-breakpad(27737): M 0000007FB1963000 0000000000000000 0000000000010000 97908C565F1A36600137BF817F34B7260 libGLESv2.so
F/google-breakpad(27737): M 0000007FB1984000 0000000000000000 0000000000009000 96B32340F96F5C7B36A213E064EA4A170 libGLESv1_CM.so
F/google-breakpad(27737): M 0000007FB1985000 0000000000000000 00000000000DE000 3FF6DC3F3EE40D14D57329976F9659FA0 libEGL.so
F/google-breakpad(27737): M 0000007FB1A85000 0000000000000000 00000000000BD000 99EF8A97B0403110EB1310DFCA88B30E0 libsqlite.so
F/google-breakpad(27737): M 0000007FB1B16000 0000000000000000 00000000004ED000 526E5203195035C799F5BC4350ACAD560 libskia.so
F/google-breakpad(27737): M 0000007FB204E000 0000000000000000 0000000000009000 A169E4DF21FFE42CAAEFDD4B52A0596E0 libcamera_metadata.so
F/google-breakpad(27737): M 0000007FB205B000 0000000000000000 0000000000042000 565807AD0B408A047E217AE578228BE40 libcamera_client.so
F/google-breakpad(27737): M 0000007FB20B5000 0000000000000000 000000000005F000 1543282A47D35BB1DA3C282F3B10784F0 libinputflinger.so
F/google-breakpad(27737): M 0000007FB2122000 0000000000000000 000000000002C000 A51AE711D22D96CDABB69C5496F38EA00 libinput.so
F/google-breakpad(27737): M 0000007FB213F000 0000000000000000 000000000009B000 1AA019977685F9900178C1E6A4CAC3F00 libgui.so
F/google-breakpad(27737): M 0000007FB21FF000 0000000000000000 000000000001B000 34F4636B14C4D36EEADF2075D9354BBE0 libui.so
F/google-breakpad(27737): M 0000007FB222E000 0000000000000000 000000000000C000 1F61D0C83115AF816BC1C5D7ECC51A8F0 libnetutils.so
F/google-breakpad(27737): M 0000007FB224C000 0000000000000000 0000000000007000 CC749FCBE7790F490545666B86CA581C0 libnativehelper.so
F/google-breakpad(27737): M 0000007FB2265000 0000000000000000 0000000000021000 B93A576B3FC3A9F3CF25F0833F6F0F500 libexpat.so
F/google-breakpad(27737): M 0000007FB2299000 0000000000000000 000000000003D000 85A829B1A4FFC17695F54A32FB8BB1A90 libandroidfw.so
F/google-breakpad(27737): M 0000007FB22E9000 0000000000000000 0000000000001000 8A668AFCA6B2F687E9296E07A1873C770 libmemtrack.so
F/google-breakpad(27737): M 0000007FB22FC000 0000000000000000 000000000000B000 4CFE19DD2D88CE936BAC58D42ABE16580 libbacktrace.so
F/google-breakpad(27737): M 0000007FB2318000 0000000000000000 0000000000038000 C42B68C501089556AA32E88DFF1572AA0 libm.so
F/google-breakpad(27737): M 0000007FB2362000 0000000000000000 00000000000C5000 FBCEFED30D5110B41E3AA12EE9D67D340 libc.so
F/google-breakpad(27737): M 0000007FB2444000 0000000000000000 00000000000E0000 480D16937E0B58DA2F082AF3DF93BC540 libc++.so
F/google-breakpad(27737): M 0000007FB2539000 0000000000000000 0000000000041000 EC033CDA48543ECDC2562064FB703B0F0 libwilhelm.so
F/google-breakpad(27737): M 0000007FB2560000 0000000000000000 0000000000190000 20937E98DB75ECE1B02C11558B5BDDCD0 libandroid_runtime.so
F/google-breakpad(27737): M 0000007FB270F000 0000000000000000 0000000000044000 7DB41D0CD4A54ADE3BAD3B8F137F85BD0 libbinder.so
F/google-breakpad(27737): M 0000007FB276F000 0000000000000000 000000000000A000 DF05A341168BA0EB42974A2D44851B770 liblog.so
F/google-breakpad(27737): M 0000007FB278A000 0000000000000000 0000000000024000 6B24E9D1126C35E344FC6860ECBE5DDA0 libutils.so
F/google-breakpad(27737): M 0000007FB27C0000 0000000000000000 0000000000016000 6FE9095ADDC6541DBA02404ECD0B66D00 libcutils.so
F/google-breakpad(27737): M 0000007FB2808000 0000000000000000 0000000000033000 B99B7E8F90D8C2480BDC5E4D135F91C20 linker64
F/google-breakpad(27737): M 0000007FB2864000 0000000000000000 0000000000001000 67D42CD6CF8AF63B3BD7F02AB02B3D0C0 linux-gate.so
F/google-breakpad(27737): -----END BREAKPAD MICRODUMP-----
W/google-breakpad(27514): ### ### ### ### ### ### ### ### ### ### ### ### ###
W/google-breakpad(27514): Chrome build fingerprint:
W/google-breakpad(27514): 70.0.3538.80
W/google-breakpad(27514): 353808050
W/google-breakpad(27514): ### ### ### ### ### ### ### ### ### ### ### ### ###
F/libc    (27514): Fatal signal 5 (SIGTRAP), code 1 in tid 27514 (wflutterexample)`